### PR TITLE
yatta!

### PR DESCRIPTION
**Pull request by nicamanas**
basic express app ready
postgresql database container created too

basic get route + manual code to add data in singular fashion

### CODE REVIEW

**Review by DavidLHW — CHANGES_REQUESTED** [review]
thanks for the pr! overall looks great and this should be enough for our use case (acting as a template to be used for other microservices)


- [ ] the express project files (package.json, package-lock, tsconfig etc) should reside inside the `./templates/express` directory instead of at the repo root
- [ ] compiled js files should not be pushed to repo (compile to `./express/dist/` and gitignored.) refer to [node-express-prisma boilerplate project](https://github.com/mcnaveen/Node-Express-Prisma-Boilerplate)

other than the above, the rest lgtm!

File: package.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+    "@prisma/client": "^4.11.0",
+    "@types/node": "^18.14.4",
+    "express": "^4.18.2",
+    "ts-node": "^10.9.1",
+    "typescript": "^4.9.5"
+  },
+  "devDependencies": {
+    "prisma": "^4.11.0"
+  }
+}
```
[the remark this answers is not in the document]
Comment (nicamanas) [reply package.json]: have shifted course files into templates/express, testing with the docker compose up postgres database, data is persisted

image

image

File: templates/express/index.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+// TODO
+var express = require("express");
+var app = express();
+var port = 5000;
+app.get("/", function (req, res) {
+    res.send("Hello World!");
+});
+app.listen(port, function () {
+    console.log("Example app listening on port ".concat(port));
+});
```
[the remark this answers is not in the document]
Comment (nicamanas) [reply templates/express/index.js]: removed the js file, updated dependencies.

File: templates/express/package.json [hunk 3]
```diff
@@ -0,0 +1,12 @@
+{
```
Comment (DavidLHW) [inline templates/express/package.json:1]: should include the following in the `package.json`:

```json
{
  "name": "carbon-marketplace-express-microservice",
  "version": "1.0.0",
  "main": "index.js",
  "scripts": {
    "start": "npm run build && node ./dist/index.js",
    "dev": "nodemon ./src/index.ts",
    "build": "tsc -p ."
  }
}
```

**Review by DavidLHW — COMMENTED** [review]
thanks for the changes! can i trouble you to add a readme on how to get started with this template project?

(we will use this template for all the microservices with express/typescript)

**Review by DavidLHW — APPROVED** [review]
CLGTM @ be86f518c4178cc29c14f70e1b9beff12b9f312c

**Pull request merged**